descriptors/ModuleDescriptor-template.json

```diff
@@ -366,7 +366,7 @@
     },
     {
       "id": "receiving",
-      "version": "1.2",
+      "version": "2.2",
```

Comment:
should be 2.0

Reply:
also we should update in requires section to new version of receiving-history interface from 3.2 to 4.0

Reply:
provided pieces API version also should be increased from 2.1 to 3.0